scripts/get-buildconfig.ps1

```diff
@@ -90,28 +90,6 @@ if ("" -eq $Tls) {
         $Tls = "schannel"
     } else {
         $Tls = "quictls"
-        try {
```

Comment:
Why is this being removed? Don't we need this logic in the future for determining whether to build OpenSSL or quictls?

Reply:
The was trying to see if we're building for openssl3. Now, we **always** build for openssl3.